org.eclipse.xtext.xtext.ui.tests/src/org/eclipse/xtext/xtext/ui/editor/hierarchy/XtextCallHierarchyBuilderTest.xtend

```diff
@@ -221,7 +222,7 @@ class XtextCallHierarchyBuilderTest extends AbstractHierarchyBuilderTest {
 					}
 					Model {
 						grammar: org.eclipse.xtext.ui.tests.editor.hierarchy.CallHierarchyBuilderTestLanguage
-						'name=ID' [295, 7]
+						'name=ID' [305, 7]
```

Comment:
You mean dig into the reason?
The build executes the test successfully: http://services.typefox.io/open-source/jenkins/job/xtext-eclipse/job/kth_20170103/lastCompletedBuild/testReport/org.eclipse.xtext.xtext.ui.editor.hierarchy/XtextCallHierarchyBuilderTest/